v.alkis.buildings.import/v.alkis.buildings.import.html

```diff
@@ -0,0 +1,60 @@
+<h2>DESCRIPTION</h2>
+
+<em>v.alkis.buildings.import</em> downloads ALKIS building data and imports them into GRASS. 
+To download the data, the federal_state(s) of the area of interest has to be given.
+Either given in a text file, comma-separated for multiple federal_states (<b>filepath</b> option).
+Or given directly as input, comma-separated for multiple federal_states (<b>federal_state</b> option).
+
+Note: Importing all data for a complete federal state could fail, due to too small RAM. To avoid this, the data can be loaded for a smaller region.
+Either by using the <b>aoi_map</b> option, or by using the <b>-r</b> flag.
+With <b>aoi_map</b>, the data are imported only for the given vector map (given in GRASS DB).
+With the <b>-r</b> flag, the data are imported only for the current set region.
+Implemented federal state options are:
```

Comment:
```suggestion
Note: Importing all data for a complete federal state could fail, due to too
small RAM. To avoid this, the data can be loaded for a smaller region, by
either by using the <b>aoi_map</b> option, or by using the <b>-r</b> flag.
With <b>aoi_map</b>, the data are imported only for the given vector map (given in GRASS DB).
With the <b>-r</b> flag, the data are imported only for the current set region.
<p>
Implemented federal state options are:
```